llvm/unittests/CAS/CASDBTest.cpp

```diff
@@ -111,6 +111,21 @@ TEST_P(CASDBTest, BlobsBig) {
     ASSERT_EQ(ID1, ID2);
     String2.append(String1);
   }
+
+  // Specifically check near 1MB for objects large enough they're likely to be
+  // stored externally in an on-disk CAS and will be near a page boundary.
+  SmallString<0> Storage;
+  const size_t InterestingSize = 1024U * 1204ULL;
```

Comment:
> 1204

typo for 1024?

Reply:
Fixed in 2c95ab3d472d00f265e48a436218935a99335500.